systemd/openqa-webui.service

```diff
@@ -1,12 +1,12 @@
 [Unit]
 Description=The openQA web UI
-Wants=openqa-setup-db.service
+Wants=openqa-setup-db.service openqa-livehandler.service openqa-websockets.service openqa-gru.service openqa-enqueue-asset-cleanup.timer openqa-enqueue-result-cleanup.timer openqa-enqueue-bug-cleanup.timer openqa-minion-restart.path
```

Comment:
Why is this needed? Would the unit otherwise restart all the required units as well?

Reply:
No, but the unit itself would be restarted if any of the other units are restarted.

Reply:
I understand. But having the other services in Wants would not fail openqa-webui.service if any of the other services fails. We might want to note this down somehow.

OTOH docs state "This is the recommended way to hook start-up of one unit to the start-up of another unit." - so maybe it is just the right thing to do. Feel free to resolve

Reply:
I started that this is acceptable behavior in the commit message. And I also read the systemd documentation which is in-line which what makes sense here specifically.